app/main.py

```diff
@@ -1,2 +1,44 @@
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.load_factor = 2 / 3
+        self.size = 0
+        self.hash_table = [[] for _ in range(self.capacity)]
+
+    def _resize(self) -> None:
+        old_hash_table = self.hash_table
+        self.capacity *= 2
+        self.hash_table = [[] for _ in range(self.capacity)]
+        self.size = 0
+
+        for slot in old_hash_table:
+            if slot:
+                self[slot[0]] = slot[1]
+
+    def _probe(self, key: any, setting: bool = True) -> int:
```

Comment:
think of a better name - such as probe_for_index. Also check type annotation for key - it should be a hashable value